pkg/pillar/scripts/device-steps.sh

```diff
@@ -377,10 +377,10 @@ if [ ! -s $CONFIGDIR/device.cert.pem ] || [ $RTC = 0 ]; then
     # Otherwise the cert may have start date in the future or in 1970
     echo "$(date -Ins -u) Check for NTP config"
     if [ -f /usr/sbin/ntpd ]; then
-        # '-p' means peer in some distros; pidfile in others
-        /usr/sbin/ntpd -q -n -p pool.ntp.org
+        # Wait until synchronized and force the clock to be set from ntp
+        /usr/sbin/ntpd -q -n -g -p pool.ntp.org
```

Comment:
I cannot find `-g` option in our version of ntpd.
```
(ns: pillar) 1401f506-5359-44e3-ae2c-e9ae3d03ed8f:/# /usr/sbin/ntpd --help
BusyBox v1.32.1 () multi-call binary.

Usage: ntpd [-dnqNwl] [-I IFACE] [-S PROG] [-k KEYFILE] [-p [keyno:N:]PEER]...

NTP client/server

        -d      Verbose (may be repeated)
        -n      Do not daemonize
        -q      Quit after clock is set
        -N      Run at high priority
        -w      Do not set time (only query peers), implies -n
        -S PROG Run PROG after stepping time, stratum change, and every 11 min
        -k FILE Key file (ntp.keys compatible)
        -p [keyno:NUM:]PEER
                Obtain time from PEER (may be repeated)
                Use key NUM for authentication
                If -p is not given, 'server HOST' lines
                from /etc/ntp.conf are used
        -l      Also run as server on port 123
        -I IFACE Bind server to IFACE, implies -l

```

Reply:
This works for me both in the eve container (where it runs) and in the host. 
Where are you running it?

Reply:
I run the command from pillar namespace. The same output is available from `eve exec pillar ntpd --help`. You command works fine, probably it is an issue in the help output.